facebook_feed_block.py

```diff
@@ -174,3 +180,23 @@ def _prepare_url(self, paging=False):
             self.paging_url = "%s&until=%d" % (self.url, self.prev_stalest)
 
         return headers
+
+    def _on_failure(self, resp, paging, url):
+        execute_retry = True
+        try:
+            status_code = resp.status_code
+            resp = resp.json()
+            err_code = resp.get('error', {}).get('code')
+            if status_code == 404 and \
```

Comment:
Minor, but since there are other changes too, let's add a comment here talking about what this error means and why we catch it. Bonus points for linking to facebook documentation for the error (in case they change it, which they have in the past...)